semgrep_output_v1.atd

```diff
@@ -1852,6 +1852,27 @@ type format_context
    is_using_registry: bool;
 }
 
+type lockfile_kind
+  <ocaml attr="deriving show, eq, yojson">
+  <python decorator="dataclass(frozen=True)">   = [
+    | PipRequirementsTxt
+    | PoetryLock
+    | PipfileLock
+    | NpmPackageLockJson
+    | YarnLock
+    | PnpmLock
+    | GemfileLock
+    | GoMod
+    | CargoLock
+    | MavenDepTree (* Not a real lockfile *)
+    | GradleLockfile
+    | ComposerLock
+    | NugetPackageLockJson
```

Comment:
Fwiw, this would be more accurate if it's `NugetPackagesLockJson` because the lockfile is `packages.lock.json`

Reply:
oh, great, thanks. will fix

Reply:
fixed!